CDCgov/prime-reportstream
# Add elr-elims topic for UP

This PR creates a new topic(`elr-elims`) for use on the universal pipeline and adds a sample sender to the org file.

Test Steps:

1. Check out and build the branch
2. add a receiver:
```
receivers:
    - name: MN_ELIMS
      organizationName: CDC-ELIMS-HL7
      topic: elr-elims
      customerStatus: active
      jurisdictionalFilter: [ "(%performerState.exists() and %performerState = 'MN') or (%patientState.exists() and %patientState = 'MN')" ]
      qualityFilter: []
      conditionFilter: []
      timing:
        operation: MERGE
        numberPerDay: 1440 # Every minute
        initialTime: 00:00
      translation: !<HL7>
        schemaName: "metadata/hl7_mapping/ORU_R01/ORU_R01-base"
        useTestProcessingMode: false
        useBatchHeaders: true
        type: "HL7"
        receivingApplicationName: ""
        receivingFacilityName: ""
      transport:
        type: SFTP
        host: sftp
        port: 22
        filePath: ./upload
        credentialName: DEFAULT-SFTP
```
4. run `./gradlew reloadSettings`
5. Send a test report using this collection:
[elims test.postman_collection.txt](https://github.com/CDCgov/prime-reportstream/files/12044120/elims.test.postman_collection.txt)
6. Verify that the request was accepted
7. Open the generated HL7 file in the local sftp folder and verify that it is HL7

## Changes
- added elr-elims topic
- add one test and expanded others
- added sender to org file

## Checklist

### Testing
